recipes/python-sonarqube-api/meta.yaml

```
@@ -0,0 +1,38 @@
+{% set name = "python-sonarqube-api" %}
+{% set version = "1.1.8" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: c254cd810ac05960864ba2077c912e15b8d35f990cda9fda9dbe20600adc4d4c
+
+build:
+  noarch: python
+  number: 0
+  script: "{{ PYTHON }} -m pip install . -vv"
+
+requirements:
+  host:
+    - python
```

The version constraint for Python in `host` and `run` should almost always match:
```suggestion
    - python >=2.7
```

---

Thank you.